### ISSUE TITLE
Please improve documentation

### ISSUE THREAD

**Issue by gauravdingolia** [cstgrid/k8comp#3]
Documentation is not straight-forward and lot of things are missing. What about multi-deployment file, config maps and other kubernetes components. Requires a better explanation.

**Comment by catalinpan** [comment]
I agree with you on this. Even the examples are hard to follow. I will work on improving the documentation. Can I know please where you struggle and I will be happy to help with some more documentation.

**Comment by catalinpan** [comment]
I have started a documentation page
https://cststack.github.io/k8comp/
I will add more things in there.

**Comment by catalinpan** [comment]
Please reopen the issue with some specific requirements regarding the documentation. Hopefully the docs provided are a good start.
